k8s/namespaces/family-public-law/fpl-case-service/fpl-case-service.yaml

```diff
@@ -18,4 +18,4 @@ spec:
       readinessTimeout: 5
       readinessPeriod: 15
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/fpl/case-service:prod-db0f179-20210901082508
+      image: hmctspublic.azurecr.io/fpl/case-service:prod-0c22f5a-20210901082313   #{"$imagepolicy": "flux-system:fpl-case-service"}
```

Comment:
Is the image name change required?

Reply:
The script update was behind the recent image update by flux.  now fixed